packages/cloud-cognitive/src/components/ModifiedTabs/ModifiedTabs.js

```diff
@@ -5,93 +5,107 @@
-        const tab = tabsRef.current.getTabAt(tabs.length);
-        if (tab & tab.tabAnchor) {
-          tab.tabAnchor.focus();
-        }
-      });
-    }
-  };
+export let ModifiedTabs = forwardRef(
+  (
+    {
+      className,
+      tabs = defaults.tabs,
+      newTabLabel = defaults.newTabLabel,
+      newTabContent = defaults.newTabContent,
+      onNewTab = defaults.onNewTab,
+      onCloseTab = defaults.onCloseTab,
+    },
+    ref
+  ) => {
+    const tabsRef = useRef();
+    const localRef = ref || tabsRef;
 
-  const handleClose = (id) => {
-    if (onCloseTab) {
-      onCloseTab(id);
-    }
-  };
+    const handleNewTab = () => {
+      if (onNewTab) {
+        onNewTab();
+        setTimeout(() => {
+          // set focus to the new tab
+          const addNewTabButton = localRef.current.querySelector(
+            `.${blockClass}__tab-new`
+          );
+          const tab = addNewTabButton?.previousSibling;
+          tab?.focus();
+        });
+      }
+    };
 
-  const tabsRef = useRef(null);
+    const handleClose = (id) => {
+      if (onCloseTab) {
+        onCloseTab(id);
+      }
+    };
 
-  return (
-    <Tabs className="modified-tabs" ref={tabsRef}>
-      {tabs.map((tab) => (
-        <Tab
-          id={tab.id}
-          key={tab.id}
-          label={
-            <ModifiedTabLabelWithClose
-              label={tab.label}
-              onClose={() => handleClose(tab.id)}
-              unsavedContent={tab.unsavedContent}
-            />
-          }
-        >
-          <div className="some-content">{tab.content}</div>
-        </Tab>
-      ))}
-      <Tab
-        id="modified-tabs__tab-new"
-        label={<ModifiedTabLabelNew label={newTabLabel} />}
-        onClick={handleNewTab}
-        onKeyUp={(ev) => ev.keyCode === 32 && handleNewTab()}
-        role="button"
-      >
-        <div className="some-content">{newTabContent}</div>
-      </Tab>
-    </Tabs>
-  );
-};
+    return (
+      <div className={cx(blockClass, className)} ref={localRef}>
+        <Tabs className="modified-tabs">
+          <TabList aria-label="Modified tab list temp aria label">
+            {tabs.map((tab) => (
+              <Tab key={tab.id}>
+                <ModifiedTabLabelWithClose
+                  label={tab.label}
+                  onClose={() => handleClose(tab.id)}
+                  unsavedContent={tab.unsavedContent}
+                />
+              </Tab>
+            ))}
+            <Tab
+              id="modified-tabs__tab-new"
+              onClick={handleNewTab}
+              onKeyUp={(ev) => ev.keyCode === 32 && handleNewTab()}
+              role="button"
+              className={`${blockClass}__tab-new`}
+            >
+              <ModifiedTabLabelNew label={newTabLabel} />
+            </Tab>
+          </TabList>
+          <TabPanels>
+            {tabs.map((tab) => (
+              <TabPanel key={tab.id} id={tab.id}>
+                <div className="some-content">{tab.content?.()}</div>
```

Comment:
Since this is in the actual component and not Storybook, I feel like we should replace this with an actual classname. But if we left this intentionally to minimize potential conflicts until we resolve in v10, then I think that’s probably okay.

Reply:
Agreed, I forgot to update this class.